Source/WebKit/UIProcess/glib/WebProcessPoolGLib.cpp

```diff
@@ -302,6 +304,31 @@ const String& WebProcessPool::generateNextAccessibilityBusName()
     return accessibilityBusName();
 }
 
+#if PLATFORM(WPE)
+OptionSet<AvailableInputTypes> WebProcessPool::availableInputTypes()
+{
+#if ENABLE(WPE_PLATFORM)
+    bool usingWPEPlatformAPI = !!g_type_class_peek(WPE_TYPE_DISPLAY);
+    if (!usingWPEPlatformAPI)
+        return AvailableInputTypes::Mouse;
```

Comment:
This is not correct, when building with wpe platform enabled, but using the old api, we should do the same was when not building with wpe-platform enabled. Old API should always do the same. I would move the new api handling inside `if (usingWPEPlatformAPI)` and returning in thar case. Then instead of `#elif`  use `#endif ` and then return Touch if touch events enabled or Mouse otherwise.